### PR TITLE
feat(components): [select-v2] Add max-collapse-tags prop

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/select-v2.md [hunk 1]
```diff
@@ -141,38 +141,39 @@ select-v2/use-valueKey
...
+| collapse-tags                                    | whether to collapse tags to a text when multiple selecting                                                                               | boolean                            | —                                                                                                         | false         |
+| collapse-tags-tooltip                            | whether show all selected tags when mouse hover text of collapse-tags. To use this, `collapse-tags` must be true                         | boolean                            | true / false                                                                                              | false         |
+| max-collapse-tags<VersionTag version="2.2.30" /> | The max tags number to be shown. To use this, `collapse-tags` must be true                                                               | number                             | —                                                                                                         | 1             |
```
Comment (jw-foss) [inline docs/en-US/component/select-v2.md:155]: Same here: https://github.com/element-plus/element-plus/pull/11378#discussion_r1106035787, once this is done, we can merge this.

Comment (kooriookami) [reply docs/en-US/component/select-v2.md]: Done. :D

**Comment by kooriookami** [comment]
@jw-foss What version tag of `max-collapse-tags` should I change?

**Comment by jw-foss** [comment]
> @jw-foss What version tag of `max-collapse-tags` should I change?

`2.3.1` will do.

**Comment by jw-foss** [comment]
@kooriookami do not use this `<VersionTag version="2.2.31" />` in the doc use the new syntax `^(2.2.31)`

**Comment by kooriookami** [comment]
All finished.

**Review by jw-foss — APPROVED** [review]

**Pull request merged**